### PR TITLE
Use Ice 3.5 for OMERO-start

### PR DESCRIPTION
**Pull request by sbesson**
To be tested by http://regions-ci.docker.openmicroscopy.org:8080/job/OMERO-start/8/

### CODE REVIEW

**Comment by joshmoore** [comment]
No objections. If this works in `regions-ci` and or `devspace-ci`, feel free to merge. I'd rather not upgrade just yet.

**Comment by sbesson** [comment]
The main downside is that a running server might have to be stopped manually before running this (since the ice to be used for stopping and starting the server are different).

**Comment by joshmoore** [comment]
In the IDR case, that's not a problem since it'll be a new docker. In the space/integration case, agreed & noted.

**Comment by sbesson** [comment]
Last commit a24eb69 pushed should fix the initial shell of the job by matching only the top-level folder using `openmicroscopy-*` rather than `openmicroscopy*` and setting up the Ice environment before stopping the running server. I would expect

```
dist/bin/omero admin stop --force-rewrite
```

to be unnecessary with this but we can clean up later.

File: home/jobs/OMERO-start/config.xml [hunk 1]
```diff
@@ -100,4 +101,4 @@ cd $SRC
     </hudson.tasks.junit.JUnitResultArchiver>
   </publishers>
   <buildWrappers/>
-</project>
+</project>
```
Comment (mtbc) [inline home/jobs/OMERO-start/config.xml]: ?

Comment (sbesson) [inline home/jobs/OMERO-start/config.xml]: Yeah... Jenkins automatic changes

Comment (sbesson) [inline home/jobs/OMERO-start/config.xml]: See e.g. https://issues.jenkins-ci.org/browse/JENKINS-25628

**Pull request merged**